packages/pwa-buildpack/src/Utilities/addImgOptMiddleware.js

```diff
@@ -70,12 +70,22 @@ https://github.com/nodejs/node-gyp#installation`
             const imageUrl = new URL(incomingUrl, imgOptConfig.baseHost);
             debug('imageUrl', imageUrl);
 
-            const optParamNames = ['auto', 'format', 'width'];
+            const optParamNames = ['auto', 'format', 'width', 'height'];
 
-            const rewritten = new URL(
-                `https://0.0.0.0/resize/${incomingQuery.width}`
-            );
+            const { width, height } = incomingQuery;
+            let rewrittenUrl = 'https://0.0.0.0/resize/';
 
+            if (width) {
+                rewrittenUrl += `${width}/`;
+            } else {
```

Comment:
This section seems unreachable given the check on 132. Can simplify and remove.

Reply:
What check are you referencing? If you request with `height=300` and no width this will error.

Reply:
https://github.com/magento-research/pwa-studio/pull/1361/files#diff-4038ce758492d17b6add3daa61c02f5eR132

It won't enter this function w/o width.

Reply:
Oh haha nice catch.